packages/api/test/e2e/tx.e2e.ts

```diff
@@ -119,16 +120,25 @@ describe('e2e transactions', () => {
           burn: null,
         },
       });
-      // transfer
-      await api.tx.genericAsset
-        .create(assetOptions)
-        .signAndSend(alice.address, async ({events, status}: SubmittableResult) => {
+
+      const sudoKey = await api.query.sudo.key();
+      const keyring = testKeyring();
+      // Lookup from keyring (assuming we have added all, on --dev this would be `//Alice`)
+      const sudoPair = keyring.getPair(sudoKey.toString());
+
+      await api.tx.sudo
+        .sudo(api.tx.genericAsset
+          .create(alice.address,
+            assetOptions
+          ))
+        .signAndSend(sudoPair, async ({events, status}: SubmittableResult) => {
           if (status.isFinalized) {
             const assetIdAfter = (await api.query.genericAsset.nextAssetId()) as AssetId;
             // expect
             expect(assetIdAfter.gt(assetIdBefore)).toBeTruthy();
-            expect(Number(assetIdAfter.toString())).toBeGreaterThan(reservedIdStart);
-            expect(Number(assetIdBefore.toString())).toBeGreaterThan(reservedIdStart);
+            // We don't know the reservedIdStart on node
```

Comment:
We can remove this test, it's kind of business logic vs. API functionality

Reply:
sure